nessie/src/test/java/org/apache/iceberg/nessie/BaseTestIceberg.java

```diff
@@ -247,6 +283,14 @@ static String metadataLocation(NessieCatalog catalog, TableIdentifier tableIdent
     return icebergOps.currentMetadataLocation();
   }
 
+  static String viewMetadataLocation(NessieCatalog catalog, TableIdentifier tableIdentifier) {
+    View view = catalog.loadView(tableIdentifier);
+    BaseView baseView = (BaseView) view;
+    ViewOperations ops = baseView.operations();
+    NessieViewOperations icebergOps = (NessieViewOperations) ops;
+    return icebergOps.current().metadataFileLocation();
```

Comment:
could also be done in a single line: `return ((BaseView) catalog.loadView(tableIdentifier)).operations().current().metadataFileLocation();`